admin/tabs/monitor/MonitorResultsModel.js

```diff
@@ -35,6 +36,21 @@ export class MonitorResultsModel {
         return this.results.filter(monitor => monitor.status === 'FAIL').length;
     }
 
+    get monitorMask() {
```

Comment:
A couple of minor comments here:  

1) I think its totally appropriate for this to just be a function in the Component file-- this is really view logic. (I would make it a function,that takes a model).

2) I think you could add a getter on the model of the form 'active', that could be symmetrical  with the other getters.
I also think we should treat 'undefined' as 'active' for now, so we can release this without being dependent on hoist core.

3) Do you need to actually do the bit with the loadModel?  Can't you just always return 'onLoad' on line 50 and have the Panel handle it for you?

Reply:
Thanks, Lee! Good points -- just tested again, and even with the new active flag, the monitors need to be run in order for the monitorResult to register the change of its monitor... not sure what the best way to handle this would be. Perhaps a label in the toolbar to display the active monitor count?